containers/dapr-dotnet/.devcontainer/devcontainer.json

```diff
@@ -23,7 +23,7 @@
 	// "forwardPorts": [],
 
 	// Ensure Dapr is running on opening the container
-	"postCreateCommand": "dapr init",
+	"postCreateCommand": "dapr uninstall && dapr init",
```

Comment:
I'd add the "all" option for uninstall; `dapr uninstall --all`.

Reply:
Otherwise that will likely do the trick.

Reply:
Done!